src/api/app/controllers/webui/project_controller.rb

```diff
@@ -655,7 +655,9 @@ def load_project_info
 
     reqs = @project.open_requests
     @requests = (reqs[:reviews] + reqs[:targets] + reqs[:incidents] + reqs[:maintenance_release]).sort!.uniq
-
+    @incoming_requests =  (reqs[:incidents] + reqs[:maintenance_release]).sort!.uniq
```

Comment:
Incoming requests are requests for which the target project is the project (`@project` in this context).

Reply:
oh, then I mixed concepts and I have the splits upside down? 
or how can I fix it?